core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpLeonardoDAOSpec.scala

```diff
@@ -42,8 +42,26 @@ class HttpLeonardoDAOSpec extends TestKit(ActorSystem("HttpLeonardoDAOSpec")) wi
         ArgumentMatchers.eq(token),
         ArgumentMatchers.eq(workspaceId),
         ArgumentMatchers.eq(s"wds-$workspaceId"),
-        ArgumentMatchers.eq("CROMWELL")
+        ArgumentMatchers.eq("CROMWELL"),
+        ArgumentMatchers.eq(None)
       )
   }
 
+  it should "call the createApp API with source workspace id" in {
+    val workspaceId: UUID = UUID.randomUUID()
+    val sourceWorkspaceId: UUID = UUID.randomUUID()
+
+    val leonardoDAO = Mockito.spy(
+      new HttpLeonardoDAO(leonardoConfig)
+    )
```

Comment:
nit: don't need a `spy` here, can just do:
```scala
val leonardoDAO = new HttpLeonardoDAO(leonardoConfig)
```
I wouldn't hold the PR for this, but if you're changing other stuff anyway might as well change this too.